frontend/packages/console-app/src/components/network-policies/network-policy-ports.tsx

```diff
@@ -1,18 +1,14 @@
 import * as React from 'react';
-import { Button } from '@patternfly/react-core';
+import { Button, FormFieldGroup, FormFieldGroupHeader } from '@patternfly/react-core';
 import { PlusCircleIcon, MinusCircleIcon } from '@patternfly/react-icons';
 import * as _ from 'lodash';
 import { useTranslation } from 'react-i18next';
 import { Dropdown } from '@console/internal/components/utils';
 import { NetworkPolicyPort } from './network-policy-model';
 
 export const NetworkPolicyPorts: React.FunctionComponent<NetworkPolicyPortsProps> = (props) => {
-  const { direction, ports, onChange } = props;
+  const { ports, onChange } = props;
```

Comment:
`direction`  seems not used anymore, if so it should be removed from props

Reply:
Fixed, thanks